zavod/zavod/exporters/peps.py

```diff
@@ -0,0 +1,111 @@
+from collections import defaultdict
+from typing import Dict, List, Any, Optional, Set, DefaultDict, NewType
+from followthemoney import model
+
+from zavod.entity import Entity
+from zavod.exporters.common import Exporter
+from zavod.util import write_json
+from zavod import helpers as h
+
+
+CountryCode = NewType("CountryCode", str)
+PositionLabel = NewType("PositionLabel", str)
+PositionId = NewType("PositionId", str)
+
+# country code -> position id -> position summary
+PositionCount = Dict[str, str | int]
+PositionSummary = DefaultDict[PositionId, PositionCount]
+Country = Dict[str, PositionSummary | str | int]
+CountryMap = DefaultDict[CountryCode, Country]
+
+# position -> country code -> country summary
+CountryCount = Dict[str, str | int]
+CountrySummary = DefaultDict[CountryCode, PositionCount]
+Position = Dict[str, CountrySummary | str | int]
+PositionMap = DefaultDict[str, Position]
```

Comment:
Is it horrible to go down this track of typing the nested dicts? This isn't used right now - it was getting hairy so I just made it  DefaultDict[str, Any] for now